### PR TITLE
draft of h/14 guide

### PR DESCRIPTION
**Pull request by nousr**

### CODE REVIEW

File: docs/laion5B_h14_back.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    - https://docs.github.com/en/repositories/working-with-files/managing-large-files/installing-git-large-file-storage
+3. run `git-lfs install`
+4. Navigate to your large storage
+    - `cd /somehwere/with/lots/of/space`
+5. Clone the repository 
+    - `git clone https://huggingface.co/datasets/laion/laion5b-h14-index`
+6. cd into repository
+    - `cd laion5b-h14-index`
+7. Download the index files with git-lfs
+    - `git-lfs fetch -I index-parts` <- *this will take quite a while*
```
Comment (rom1504) [inline docs/laion5B_h14_back.md:15]: let's use aria2x -x 16 instead

File: docs/laion5B_h14_back.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    - `git-lfs checkout index-parts/*` <- *this will take quite a while*
+8. Combine the index parts using the following command
+    - `clip_retrieval index_combiner --input_folder "the/indices" --output_folder "output"`
+
+9. Now repeat steps 5-7 with the following metadata repos
```
Comment (rom1504) [inline docs/laion5B_h14_back.md:20]: let's provide the exact commands / make it a script

Comment (nousr) [reply docs/laion5B_h14_back.md]: done

File: docs/laion5B_h14_back.md (hunk 1 above)
Comment (nousr) [reply docs/laion5B_h14_back.md]: done

**Pull request merged**